Warning when compiling for ESP32-CAM (AI Thinker)

<!-- Thanks for reporting a bug for this project. READ THIS FIRST:

- Provide as many details as possible. Simply saying "X gives bug" or "X gives error" is not enough!
- Paste logs, configuration sample and code into the backticks (```).
- Read through the template carefully and fill out all missing details.
- Please also search for similar issues in this issue tracker first and read through the ESPHome FAQ.

DO NOT DELETE ANY TEXT from this template! Otherwise the issue may be closed without a comment.
-->

**Operating environment/Installation (Hass.io/Docker/pip/etc.):**
<!--
Please provide details about your environment below this line. -->

**ESP (ESP32/ESP8266, Board/Sonoff):**
ESP32S AI Thinker

**Affected component:**
Camera


**Description of problem:**
I get a warning when compiling under ESPHome 1.14.3, though the camera seems to work

**Problem-relevant YAML-configuration entries:**
```
substitutions:
  # Modify variables based on your settings
  hostname: 'camera_kitchen'

esphome:
  name: $hostname
  platform: ESP32
  board: esp32dev

wifi:
  ssid: "SSID"
  password: "PW"

  # Enable fallback hotspot (captive portal) in case wifi connection fails
  ap:
    ssid: "Camera Kitchen Fallback Hotspot"
    password: "PW"

#captive_portal:

# Enable logging
logger:

# Enable Home Assistant API
api:

ota:
  password: "PW"

# ESP32-CAM
esp32_camera:
  external_clock:
    pin: GPIO0
    frequency: 20MHz
  i2c_pins:
    sda: GPIO26
    scl: GPIO27
  data_pins: [GPIO5, GPIO18, GPIO19, GPIO21, GPIO36, GPIO39, GPIO34, GPIO35]
  vsync_pin: GPIO25
  href_pin: GPIO23
  pixel_clock_pin: GPIO22
  power_down_pin: GPIO32

  name: kitchen
  # ... Frame Settings

# Flashlight
output:
  - platform: gpio
    pin: GPIO4
    id: gpio_4
light:
  - platform: binary
    output: gpio_4
    name: $hostname light
```

**Logs (if applicable):**
<!--
Please copy the debug log here. If possible, also connect to the ESP over USB and copy those logs into the backticks.
-->
During compilation I see this warning
```
Compiling /data/camera_hallway/.pioenvs/camera_hallway/src/esphome/components/light/addressable_light.cpp.o
src/esphome/components/esp32_camera/esp32_camera.cpp: In member function 'virtual void esphome::esp32_camera::ESP32Camera::dump_config()':
src/esphome/components/esp32_camera/esp32_camera.cpp:54:10: warning: enumeration value 'FRAMESIZE_QXGA' not handled in switch [-Wswitch]
   switch (this->config_.frame_size) {
          ^
src/esphome/components/esp32_camera/esp32_camera.cpp:54:10: warning: enumeration value 'FRAMESIZE_INVALID' not handled in switch [-Wswitch]
```

**Additional information and things you've tried:**

I've tried going back to 1.13.6 because the 1.14.3 is very unstable here. That gave other problems, which are described in a seperate issue.


<!-- LEAVE THIS LINE AS-IS AND DON'T DELETE IT, OTHERWISE THE ISSUE WILL BE CLOSED AUTOMATICALLY. -->

--- a/esphome/components/esp32_camera/esp32_camera.cpp
+++ b/esphome/components/esp32_camera/esp32_camera.cpp
@@ -85,6 +85,8 @@ void ESP32Camera::dump_config() {
     case FRAMESIZE_UXGA:
       ESP_LOGCONFIG(TAG, "  Resolution: 1600x1200 (UXGA)");
       break;
+    default:
+      break;
   }
 
   if (this->is_failed()) {

